graylog2-web-interface/src/components/common/SourceCodeEditor.jsx

```diff
@@ -11,6 +10,7 @@ import fetch from 'logic/rest/FetchProvider';
 import { Button, ButtonGroup, ButtonToolbar, OverlayTrigger, Tooltip } from 'components/graylog';
 import PipelineRulesMode from 'components/rules/mode-pipeline';
 
+import AceEditor from './AsyncAceEditor';
```

Comment:
Could we just use `React.lazy` ?

```suggestion
const AceEditor = React.lazy(() => import('react-ace-builds'));
```

Reply:
https://reactjs.org/docs/code-splitting.html#reactlazy

> This will automatically load the bundle containing the `react-ace-builds` when this component is first rendered.

Reply:
Sure, I tried it a couple of weeks ago and it works. Not in this PR though. 

The benefit is close to zero though, as `loadable-component` is doing a couple of other things (loading indicator, error/fallback component) already and its size is pretty small. Nevertheless we can do it at some point in another PR, just to get rid of a dependency.